### ISSUE TITLE
[en] verb repetition false positive

### ISSUE THREAD

**Issue by donnerpeter** [languagetool-org/languagetool#6568]
`If this map maps one or more keys to the specified value...`

**Comment by 1371131436** [comment]
This problem can be solved by modifying the rule REPEATED_VERBS in grammar.xml, and I am considering a   
good antipattern for this case.

**Comment by MikeUnwalla** [comment]
Possibly, you can add chunks to the pattern. For the test sentence, the chunker gives a correct analysis. (I used an LT snapshot from 6 April for this screenshot):

![image](https://user-images.githubusercontent.com/5816160/168417731-5185026f-b9a4-4051-b901-7a2ccadc3db9.png)

Another option is to make the disambiguation better so that it gives the correct POS (map/NN and maps/VBZ).

**Comment by 1371131436** [comment]
@MikeUnwalla Yes, I have solved this problem by adding chunks, and I am making a pull request to solve this issue.

**Comment by MikeUnwalla** [comment]
Great, thank you. Just beware that the chunker isn't always correct.

I'll leave it to the regular maintainers to deal with your pull request.
